lib/Data/Validate/Sanctions/Fetcher.pm

```diff
@@ -0,0 +1,121 @@
+package Data::Validate::Sanctions::Fetcher;
+
+use strict;
+use warnings;
+
+use LWP::UserAgent;
+use Text::CSV;
+use XML::Simple qw(:strict);
+use DateTime::Format::Strptime;
+use IO::Uncompress::Unzip qw(unzip $UnzipError);
+use List::Util qw/uniq/;
+use Mojo::UserAgent;
+use Try::Tiny;
+
+my $config = {
+    'OFAC-SDN' => {
+        descrition => 'TREASURY.GOV: Specially Designated Nationals List with a.k.a included',
+        url    => 'https://www.treasury.gov/ofac/downloads/sdn_xml.zip',    #let's be polite and use zippped version of this 7mb+ file
+        parser => \&_ofac_xml_zip,
+    },
+    'OFAC-Consolidated' => {
+        description => 'TREASURY.GOV: Consolidated Sanctions List Data Files',
+        url         => 'https://www.treasury.gov/ofac/downloads/consolidated/consolidated.xml',
+        parser      => \&_ofac_xml,
+    },
+    'HMT-Sanctions' => {
+        description => 'GOV.UK: Financial sanctions: consolidated list of targets',
+        url         => 'http://hmt-sanctions.s3.amazonaws.com/sanctionsconlist.csv',
+        parser      => \&_hmt_csv,
+    },
+};
```

Comment:
consider split this module into 3 or 2 submodules, as they have completely diffrent input (xml, zip, csv) + have top-level interface like

```
my $results = fecth('OFAC-SDN', 'OFAC-Consolidated', 'HMT');
```

+ tests (mocked results) for each one, of course :)